src/features/listing-builder/components/ListingBuilder/ListingPayments.tsx

```diff
@@ -386,21 +401,87 @@ export const ListingPayments = ({
               </Text>
             </Box>
           )}
-          <FormControl pos="relative" isRequired>
+          <FormControl pos="relative">
             <ListingFormLabel htmlFor="token">Select Token</ListingFormLabel>
             <InputGroup>
-              <InputLeftElement ml={2}>
-                {searchState.tokenImage ? (
-                  <Image
-                    w={'1.6rem'}
-                    alt={searchState.tokenImage}
-                    rounded={'full'}
-                    src={searchState.tokenImage}
-                  />
-                ) : (
-                  <SearchIcon color="gray.300" />
+              {token && token !== 'USDC' && (
+                <>
+                  <InputLeftElement ml={2}>
+                    {searchState.tokenImage === null ? (
+                      <Image
+                        w={'1.6rem'}
+                        alt={
+                          tokenList.find((t) => t.tokenSymbol === token)
+                            ?.tokenName
+                        }
+                        rounded={'full'}
+                        src={
+                          tokenList.find((t) => t.tokenSymbol === token)?.icon
+                        }
+                      />
+                    ) : searchState.tokenImage !== null &&
+                      searchState.tokenImage !== '' ? (
+                      <Image
+                        w={'1.6rem'}
+                        alt={searchState.tokenImage}
+                        rounded={'full'}
+                        src={searchState.tokenImage}
+                      />
+                    ) : (
+                      <SearchIcon color="gray.300" />
+                    )}
+                  </InputLeftElement>
+                </>
+              )}
+              {token === 'USDC' &&
+                !window.location.href.includes('edit') &&
```

Comment:
not a good practise
pre load through the form state

Reply:
Do you think I should put USDC with its icon by default? or should I keep the placeholder as default?

Reply:
@a20hek made some changes